themes/MUG/invenio.cfg

```diff
@@ -9,19 +9,33 @@ https://inveniordm.docs.cern.ch/reference/configuration/.
 
 from datetime import datetime
 from invenio_i18n import lazy_gettext as _
-
+from invenio_oauthclient.contrib.keycloak import KeycloakSettingsHelper
+from invenio_rdm_records.services.components import (
+    DefaultRecordsComponents as RDMDefaultRecordsComponents,
+)
+from invenio_records_lom.services.components import (
+    DefaultRecordsComponents as LOMDefaultRecordsComponents,
+)
+from invenio_records_marc21.services.components import (
+    DefaultRecordsComponents as Marc21DefaultRecordsComponents,
+)
+from invenio_global_search.components import (
+    RDMToGlobalSearchComponent,
+    LOMToGlobalSearchComponent,
+    Marc21ToGlobalSearchComponent,
+)
```

Comment:
remove these since its already defined in the [invenio-override](https://github.com/sharedRDM/instance/blob/ebe2c248484fb6e37359088871401ad1c6646467/themes/MUG/invenio.cfg#L13)

Reply:
ill keep the keylock as i think is used from you..